src/smc-webapp/frame-editors/x11-editor/launcher.tsx

```diff
@@ -9,7 +9,17 @@ const { Icon } = require("r_misc");
 
 import { Actions } from "./actions";
 
-const APPS = {
+interface IAPPS {
+  [k: string]: {
+    icon: string;
+    desc: string;
+    label?: string;
+    command?: string;
```

Comment:
When is a button not going to have a command?

Reply:
sometimes the key is used, look at the code at the bottom right there:  https://github.com/sagemathinc/cocalc/blame/66523964ee8d6cdfd1bddd7ded8ff64b1ee00ab5/src/smc-webapp/frame-editors/x11-editor/launcher.tsx#L241